lib/pbench/server/api/user_manager.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+import tempfile
+from flask_bcrypt import Bcrypt
+from pbench.server.api.resources.database import Database
+
+
+class UserManager:
+    def __init__(self, app, server_config):
+        self.app = app
+        self.server_config = server_config
+        self.bcrypt = Bcrypt(app)
+        self.db = Database()
+
+        try:
+            # Initialize the pbench databases
+            self.db_session = self.db.init_db(self.server_config)
+        except Exception as e:
+            print(
+                f"Some Error occurred while initializing the database engine: ERROR: {e.__traceback__}"
+            )
+            sys.exit(1)
+
+        # Initialize the user sessions parameters
+        self.app.config[
+            "SESSION_TYPE"
+        ] = "filesystem"  # This need to be changed to redis
+        self.app.config["SESSION_FILE_DIR"] = tempfile.mkdtemp()
+
+        # set app secret key
+        self.app.secret_key = os.getenv("SECRET_KEY", "my_precious")
```

Comment:
This is being done in anticipation of other components that need the key accessing it through the `App` object. Seems this is pointless when they're not doing so!

Reply:
We are using flask sessions and `app.secret_key` is used by the flask session to encrypt the session data before writing on the redis instance or any other data-store